input/examples/practitioner-bobrester-bob.xml

```diff
@@ -8,12 +8,12 @@
         <type>
             <coding>
                 <system value="http://terminology.hl7.org.au/CodeSystem/v2-0203"/>
-                <code value="UPIN"/>
+                <code value="EI"/>
             </coding>
-            <text value="Medicare Provider Number"/>
+            <text value="Employee number"/>
         </type>
-        <system value="http://ns.electronichealth.net.au/id/medicare-provider-number"/>
-        <value value="1234561A"/>
+        <system value="http://bobrestermedical.example.com/id/employee-id"/>
```

Comment:
EIs need assigner to be conformant

Reply:
done

Reply:
Done; removed Medicare Prescriber Number